amethyst_renderer/src/renderer.rs

```diff
@@ -91,6 +94,7 @@ impl Renderer {
         use gfx::Device;
         #[cfg(feature = "opengl")]
         use glutin::GlContext;
+        let start = Instant::now();
```

Comment:
We should check against `Instant` from last time this funtion was called.
Store it in struct and update after waiting.